### DISCUSSION TITLE
RStudio, Quarto and presentations: text out of slide

### DISCUSSION THREAD

**Discussion by lorenzoFabbri** [quarto-dev/quarto-cli#955] [Q&A] [answered]
I am trying to use RStudio to create a Quarto presentation
```
---
title: "Statistical Rethinking - Chapter 7"
author: "Lorenzo Fabbri"
format: revealjs
editor: visual
editor_options: 
  chunk_output_type: console
---
```
The problem I am getting is that once compiled, most of the text goes outside the margin of some slides (both on the bottom and on the left). Any suggestion?

As an example:
```
## Log-probability Score

It seems that it boils down to understanding what is this score and where it comes from.

It is useful to look at how the `lppd` is actually computed (skipping the details):

-   `logprob <- rethinking::sim(model, ll = TRUE, n = 1e4)`

-   The key here is the `ll = TRUE`. We are computing the log-likelihood of the fitted model: `dnorm(x = y, mean = predict(model), sd = …)`

-   What we are doing boils down to computing the probability density of each observation `i`

-   This is indeed what Information Theory tells us: what matters is the joint probability of the data$$S(q) = \prod_i q_i = \sum_i log(q_i).$$
```

I am using RStudio 2022.06.0-daily+406 "Spotted Wakerobin" Daily (5e9b6b93c1665966df7f5acca1cb9590f4e004e5, 2022-05-15) for Ubuntu Jammy
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) QtWebEngine/5.12.8 Chrome/69.0.3497.128 Safari/537.36.

**Comment by jjallaire** [comment, marked as answer]
Thanks for reporting! This was a CSS oversight, fixed here: https://github.com/quarto-dev/quarto-cli/commit/277d1df80082d99ad0f418b0d0ee071bbe48adf9

Note that you still might need to make your text smaller or scrolling to fit it on one slide: https://quarto.org/docs/presentations/revealjs/#content-overflow
